GVFS/GVFS/CommandLine/ConfigVerb.cs

```diff
@@ -10,18 +11,32 @@ public class ConfigVerb : GVFSVerb.ForNoEnlistment
         private const string ConfigVerbName = "config";
         private LocalGVFSConfig localConfig;
 
+        [Option(
+            'l',
+            "list",
+            Required = false,
+            HelpText = "Show all settings")]
+        public bool List { get; set; }
+
+        [Option(
+            'd',
+            "delete",
+            Required = false,
+            HelpText = "Delete specified setting")]

```

Comment:
Should this read something like "Name of setting to delete"? Does that end up looking confusing in the help text?